src/Helpers/countRating.js

```diff
@@ -1,10 +1,10 @@
 export default (ratings) => {
-  let users = 0;
-  let rates = 0;
-  ratings.map((single) => {
-    rates += single.rating * single.users;
-    users += single.users;
-    return true;
-  });
-  return rates / users;
+  const x = ratings.map(item => ({
```

Comment:
@Ramadhan0 @nkpremices why is this happening on the front-end?

Reply:
Here I was calculating the rating percentage in order to fill the rating stars @MemunaHaruna